src/main/io/displayport_msp_dji_compat.c

```diff
@@ -142,12 +142,13 @@ uint8_t getDJICharacter(uint8_t ch, uint8_t page)
 
         case SYM_MAH:
             return DJI_SYM_MAH;
-
+/*
```

Comment:
I believe symbols 0x7D and 0x7E are Km and Miles respectively.

![image](https://github.com/user-attachments/assets/1f36e9ad-1b8d-4f63-8b30-463a9bb2f110)

Reply:
What line are you referring to? The efficiency ones are not available in the DJI/BF font. 

The distance 0x&d and 0x7E are used in SYM_KM, SYN_MI

Reply:
Ah yeah, it's Ah/Km etc. My bad.